addons/calendar/models/calendar.py

```diff
@@ -842,7 +842,7 @@ def _compute_is_highlighted(self):
     ], string='By day')
     final_date = fields.Date('Repeat Until')
     user_id = fields.Many2one('res.users', 'Owner', states={'done': [('readonly', True)]}, default=lambda self: self.env.user)
-    partner_id = fields.Many2one('res.partner', string='Responsible', related='user_id.partner_id', readonly=True)
+    partner_id = fields.Many2one('res.partner', string='Scheduled by', related='user_id.partner_id', readonly=True)
```

Comment:
You can also rework this commit message based on the guidelines and the example I gave you.